### PR TITLE
Basic test to use `capsule-installer` and assure proper repo is used

### PR DESCRIPTION
**Pull request by cswiii**

### CODE REVIEW

File: tests/foreman/cli/test_capsule_installer.py [hunk 1]
```diff
@@ -4,6 +4,22 @@
 
 
 class TestCapsuleInstaller(CLITestCase):
+    def capsule_installer_basic_test(self):
+        """@Test: perform a basic install of capsule.
+
+        @Feature: Capsule Installer
+
+        @Steps:
+        1. Assure your target capsule has ONLY the Capsule repo enabled. In
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: There are trailing withe spaces on this line

File: tests/foreman/cli/test_capsule_installer.py [hunk 2]
```diff
@@ -4,6 +4,22 @@
...
 class TestCapsuleInstaller(CLITestCase):
+    def capsule_installer_basic_test(self):
+        """@Test: perform a basic install of capsule.
+
+        @Feature: Capsule Installer
+
+        @Steps:
+        1. Assure your target capsule has ONLY the Capsule repo enabled. In 
+        other words, the Satellite repo itself is not enabled by default.
+        2. attempt to perform a basic, functional install the capsule using
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: There are trailing withe spaces on this line

File: tests/foreman/cli/test_capsule_installer.py [hunk 3]
```diff
@@ -4,6 +4,22 @@
 
 
 class TestCapsuleInstaller(CLITestCase):
+    def capsule_installer_basic_test(self):
+        """@Test: perform a basic install of capsule.
+
+        @Feature: Capsule Installer
+
+        @Steps:
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Format steps like this:

```
@Steps:

1. this is a really long, long, long, long, long, long, long, long, line
   the line continuation should be aligned this way
2. Second item of the list
```

File: tests/foreman/cli/test_capsule_installer.py [hunk 4]
```diff
@@ -4,6 +4,56 @@
...
+        1. Assure your target capsule has ONLY the Capsule repo enabled. In
+           other words, the Satellite repo itself is not enabled by default.
+
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: No blank line needed

File: tests/foreman/cli/test_capsule_installer.py [hunk 5]
```diff
@@ -4,6 +4,56 @@
...
+        2. attempt to perform a basic, functional install the capsule using
+           `capsule-installer`.
+
+        @Assert: product is installed
+
+        @Status: Manual
+        """
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Please add a blank line before """

File: tests/foreman/cli/test_capsule_installer.py [hunk 6]
```diff
@@ -4,6 +4,56 @@
...
+        @Feature: capsule Installer
+
+        @Steps:
+        1. Install capsule-installer with the '--qpid-router=true` flag
+
+        @Assert: Capsule installs correctly and qpid functionality is
+        enabled.
+
+        @Status: Manual
+        """
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Please add a blank line before """

File: tests/foreman/cli/test_capsule_installer.py [hunk 7]
```diff
@@ -4,6 +4,56 @@
 
 
 class TestCapsuleInstaller(CLITestCase):
+    def capsule_installer_basic_test(self):
+        """@Test: perform a basic install of capsule.
+
+        @Feature: Capsule Installer
+
+        @Steps:
+        1. Assure your target capsule has ONLY the Capsule repo enabled. In
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Add a blank line above

File: tests/foreman/cli/test_capsule_installer.py [hunk 8]
```diff
@@ -4,6 +4,56 @@
...
+
+        @Assert: Capsule installs correctly and qpid functionality is
+        enabled.
+
+        @Status: Manual
+        """
+
+    def capsule_installer_reverse_proxy(self):
+        """
+        @Test:  assure the --reverse-proxy flag can be used in
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Move `@Test` right after `"""`

File: tests/foreman/cli/test_capsule_installer.py [hunk 9]
```diff
@@ -4,6 +4,56 @@
...
+        @Feature: capsule Installer
+
+        @Steps:
+        1. Install using the '--reverse-proxy=true' flag
+
+        @Assert: Capsule installs correctly and functionality is
+        enabled.
+
+        @Status: Manual
+        """
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Please add a blank line before """

File: tests/foreman/cli/test_capsule_installer.py [hunk 10]
```diff
@@ -4,6 +4,56 @@
...
+    def capsule_installer_reverse_proxy(self):
+        """
+        @Test:  assure the --reverse-proxy flag can be used in
+        capsule-installer to enable katello-agent functionality via
+        remote clients
+
+        @Feature: capsule Installer
+
+        @Steps:
+        1. Install using the '--reverse-proxy=true' flag
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Please add a blank line above

File: tests/foreman/cli/test_capsule_installer.py [hunk 11]
```diff
@@ -4,6 +4,56 @@
...
+
+    def capsule_installer_qpid(self):
+        """@Test: assure the --qpid-router flag can be used in
+        capsule-installer to enable katello-agent functionality via
+        remote clients
+
+        @Feature: capsule Installer
+
+        @Steps:
+        1. Install capsule-installer with the '--qpid-router=true` flag
```
Comment (elyezer) [inline tests/foreman/cli/test_capsule_installer.py]: Please add a blank line above

**Comment by elyezer** [comment]
ACK, pending some formatting nitpick comments

**Comment by Ichimonji10** [comment]
APT

**Pull request merged**